tests/Responsive.test.js

```diff
@@ -7,6 +8,8 @@ import {
 import React from 'react';
 import mockContextContent from './__fixtures__/mockContextContent';
 
+const mockProps = { breakpoints, breakpointsMax };
```

Comment:
Could we name this props instead of mockProps I think using the prefix mock props everywhere in the tests doesnt make the legibility better.
We already know we are on `.test.js` file so why name everything with prefix mock?

Reply:
I agree with this change, we could remove the preffix "mock" for everything

Reply:
right! I'll do these changes.

Reply:
done!